### ISSUE TITLE
Automatic Reprioritization from Future to Active at [today - 30days]

### ISSUE THREAD

**Issue by tayamueller** [ParabolInc/action#213]
**After #1212, #1211, and #296:**

## Issue - Enhancement

Future cards will eventually be "pulled over" to become Active. Let's help our users with the frequently-observed behavior where cards "languish" or "rot" in the Future column because they're forgotten.

### Acceptance Criteria (optional)
Users can:
   - Expect that when a "Future" card's automatic due date [per #1212] reaches the state of [today - 30days] the card will automatically move over to the bottom of their "Active" column.

**Estimated effort:** X points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by jordanh** [comment]
From issue #613:

## Issue - Enhancement

When you create a future card, you can optionally add a "re-raise in meeting" date. When that date expires, raise the card as an agenda item within the meeting.

(N.B. consider a single field "re-raise on" or "due-on" to support two kinds of time-sensitive tasks)

Very related ideas.

**Comment by jordanh** [comment]
Closing in favor of new design/issue

**Comment by jordanh** [comment]
Stuck on dates in data model

**Comment by jordanh** [comment]
This needs estimated effort. Removing from sprint board for now.

**Comment by jordanh** [comment]
Iceboxing

**Comment by tayamueller** [comment]
validation: https://docs.google.com/document/d/1CSfl9u6VXaCG7gw3AviLCeEs9M-rUQ_lqXdYkKlZLaY/edit

> I have a terrible memory, so something like remembering to pull the template card over at the beginning of the month--it would be a week later that I would be like oh right I forgot to pull that card over. I would probably need like a card in my Inbox like a reminder every month: first of the month! pull the template card over!
